_config.yml

```diff
@@ -23,6 +23,10 @@ kramdown:
   syntax_highlighter: rouge
   toc_levels: 2..3
 
+# https://jekyllrb.com/docs/configuration/liquid/
+liquid:
+  error_mode: strict
```

Comment:
The only concern I have is that these errors may happen in external sources (eg reference YAMLdocs or Markdown we pull in), and while these should be addressed, we may not be able to do that immediately 🤔 


I'm surprised that the format string in compose didn't show up in the rendered output though; ISTR that liquid would just ignore the "invalid" liquid templates, and leave them as-is (so the warnings could be ignored if it wasn't really a liquid template)

Reply:
> The only concern I have is that these errors may happen in external sources (eg reference YAMLdocs or Markdown we pull in), and while these should be addressed, we may not be able to do that immediately 🤔

We have the same issue with htmlproofer I guess, not sure what we can do about it except if upstream run the same validation steps like we do in buildx repo: https://github.com/docker/buildx/pull/1218